src/expr/src/scalar/func.rs

```diff
@@ -10,9 +10,10 @@
 use std::cmp::{self, Ordering};
 use std::convert::{TryFrom, TryInto};
 use std::fmt;
+use std::ops::Add;
```

Comment:
This is a little odd. It's a bit more idiomatic to just use the `+` operator rather than importing the underlying trait.

Reply:
ok yeah I was wondering about this thanks!